### PR TITLE
Remove yarn

### PR DESCRIPTION
**Pull request by ribose-jeffreylau**
@strogonoff There are some issues with the Dockerfiles, maybe you could take a look and see what can be done?  Thanks!

### CODE REVIEW

**Comment by ribose-jeffreylau** [comment]
@strogonoff Running `dev` and changing some sources triggers immediate rerendering of the Paneron window so I guess `dev` works?
However, I think I need some help getting the Dockerfiles work.

**Comment by strogonoff** [comment]
@ribose-jeffreylau  That’s awesome. We can ignore Dockerfiles for now…

File: Dockerfile [hunk 1]
```diff
@@ -30,18 +33,19 @@ USER paneron
...
+RUN pnpm install
 
-RUN npx electron-rebuild
-# RUN yarn add typescript-language-server "typescript@4.2.2"
+# RUN pnpm --package=@electron/rebuild dlx electron-rebuild -v $(jq -r .devDependencies.electron < package.json)
 
 # see https://github.com/electron/electron/issues/17972
 USER root
-RUN chown root ${project_path:?}/node_modules/electron/dist/chrome-sandbox
-RUN chmod 4755 ${project_path:?}/node_modules/electron/dist/chrome-sandbox
```
Comment (strogonoff) [inline Dockerfile:44]: Per comments in the file, `Dockerfile` and `tsserver.Dockerfile` are intended to be line-by-line equivalent to aid subsequent refactoring. I’ll fix this.

Comment (ribose-jeffreylau) [reply Dockerfile:44]: @strogonoff Thanks for looking into it.  This is the part that I got stuck in, having no idea how it's supposed to work in the first place.  Was trying to remove things that I thought might not be needed for various reasons...

**Pull request merged**